typewise_alert.py

```diff
@@ -40,7 +49,7 @@ def print_mail(message):
       print(f'Hi, the temperature is {message}')
   return print_mail(text_msg.get(breachType))
 
-message_target={
+message_to_target={
```

Comment:
If we make a mistake/bug and send_to_controller against 'to_console', the tests will still pass.
Same if frame_email function misses the 'To' field.
That means even after your tests pass, you will need to rely on an integrated environment to prove your code.
Strengthen your tests to prove your code without having to integrate with 3rd parties.